src/Jackalope/Lock/LockManager.php

```diff
@@ -260,7 +258,8 @@ public function createLockInfo()
      */
     public function logout()
     {
-        foreach ($this->locks as $path => $lock) {;
+        foreach ($this->locks as $path => $lock) {
+            ;
```

Comment:
don't you eliminate superflous `;` in general? it would probably make sense even though its not strictly code style but more sort of an error.